addon/components/bourbon-search-select-field.js

```diff
@@ -104,7 +104,11 @@ export default Component.extend(SelectMixin, {
       e.preventDefault();
       // if valid option availble select first option upon enter
       if (!this.get('noResults') &&  this.get('searchList').length > 0) {
-        this.set('activeOption', 0);
+        // only select the first option if we have not keyed down
+        // to a dfferent selection
+        if (this.get('activeOption') === null) {
```

Comment:
Any chance this would come back `undefined` instead of null?

Reply:
I don't think so.  It gets default to null and set as null all over.  I guess I could check for `undefined` just in case.